Edu-platform_Gateway/ocelot.Production.json

```diff
@@ -1,207 +1,151 @@
-        {
-          "Host": "localhost",
-          "Port": 8040
-        }
-      ],
-      "DownstreamPathTemplate": "/api/File/{fileId}"
-    },
-    // Forum Web API
-    {
-      "UpstreamPathTemplate": "/api/Forum",
-      "UpstreamHttpMethod": [ "Get", "Post", "Put" ],
-      "DownstreamScheme": "https",
-      "DownstreamHostAndPorts": [
-        {
-          "Host": "localhost",
-          "Port": 8050
-        }
-      ],
-      "DownstreamPathTemplate": "/api/Forum"
-    },
-    {
-      "UpstreamPathTemplate": "/api/Forum/{forumId}",
-      "UpstreamHttpMethod": [ "Get", "Delete" ],
-      "DownstreamScheme": "https",
-      "DownstreamHostAndPorts": [
-        {
-          "Host": "localhost",
-          "Port": 8050
+          "Port": 8010
         }
       ],
-      "DownstreamPathTemplate": "/api/Forum/{forumId}"
+      "DownstreamPathTemplate": "/api/Claim",
+      "AuthenticationOptions": {
+        "AuthenticationProviderKey": "Bearer",
+        "AllowedScopes": []
+      } //,
+      //"RouteClaimsRequirement": {
+      //  "roles": "admin"
+      //}
     },
-    // AIBot Web API
+    // Claim {id} Identity Web API
     {
-      "UpstreamPathTemplate": "/api/AIBot",
-      "UpstreamHttpMethod": [ "Get", "Post", "Put" ],
+      "UpstreamPathTemplate": "/api/Claim/{id}",
+      "UpstreamHttpMethod": [ "Get", "Put", "Delete" ],
       "DownstreamScheme": "https",
       "DownstreamHostAndPorts": [
         {
           "Host": "localhost",
-          "Port": 8060
+          "Port": 8010
         }
       ],
-      "DownstreamPathTemplate": "/api/AIBot"
+      "DownstreamPathTemplate": "/api/Claim/{id}",
+      "AuthenticationOptions": {
+        "AuthenticationProviderKey": "Bearer",
+        "AllowedScopes": []
+      }
     },
+    // Role Identity Web API
     {
-      "UpstreamPathTemplate": "/api/AIBot/{aibotId}",
-      "UpstreamHttpMethod": [ "Get", "Delete" ],
+      "UpstreamPathTemplate": "/api/Role",
+      "UpstreamHttpMethod": [ "Get", "Post" ],
       "DownstreamScheme": "https",
       "DownstreamHostAndPorts": [
         {
           "Host": "localhost",
-          "Port": 8060
+          "Port": 8010
         }
       ],
-      "DownstreamPathTemplate": "/api/AIBot/{aibotId}"
+      "DownstreamPathTemplate": "/api/Role",
+      "AuthenticationOptions": {
+        "AuthenticationProviderKey": "Bearer",
+        "AllowedScopes": []
+      }
     },
-    // Analytics Web API
+    // Role {id} Identity Web API
     {
-      "UpstreamPathTemplate": "/api/Analytics",
-      "UpstreamHttpMethod": [ "Get", "Post", "Put" ],
+      "UpstreamPathTemplate": "/api/Role/{id}",
+      "UpstreamHttpMethod": [ "Get", "Put", "Delete" ],
       "DownstreamScheme": "https",
       "DownstreamHostAndPorts": [
         {
           "Host": "localhost",
-          "Port": 8070
+          "Port": 8010
         }
       ],
-      "DownstreamPathTemplate": "/api/Analytics"
+      "DownstreamPathTemplate": "/api/Role/{id}",
```

Comment:
@Nazaziik let's use small letter

Reply:
https://learn.microsoft.com/en-us/aspnet/web-api/overview/web-api-routing-and-actions/attribute-routing-in-web-api-2

Reply:
Or did you find different convention rules?

Reply:
Fixed